ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/140__service_mesh.robot

```diff
@@ -0,0 +1,82 @@
+*** Settings ***
+Documentation       Test Cases to verify Service Mesh integration
+
+Library             Collections
+Resource            ../../../../Resources/OCP.resource
+Resource            ../../../../Resources/ODS.robot
+Suite Setup         Suite Setup
+Suite Teardown      Suite Teardown
+
+
+*** Variables ***
+${OPERATOR_NS}                              ${OPERATOR_NAMESPACE}
+${RHOAI_OPERATOR_DEPLOYMENT_NAME}           rhods-operator
+${DSCI_NAME}                                default-dsci
+${SERVICE_MESH_OPERATOR_NS}                 openshift-operators
+${SERVICE_MESH_CR_NS}                       istio-system
+${SERVICE_MESH_OPERATOR_DEPLOYMENT_NAME}    istio-operator
+
+${IS_PRESENT}                           0
+${IS_NOT_PRESENT}                       1
+
+
+*** Test Cases ***
+Validate Service Mesh State Managed
+    [Documentation]    The purpose of this Test Case is to validate Service Mesh integration
+    [Tags]    Operator    Smoke    ODS-2526    ServiceMesh-Managed
+
+    Is Resource Present    ServiceMeshControlPlane    data-science-smcp    ${SERVICE_MESH_CR_NS}    ${IS_PRESENT}
+    Check If Pod Exists    istiod    ${SERVICE_MESH_CR_NS}
+
+Validate Service Mesh State Unmanaged
+    [Documentation]    The purpose of this Test Case is to validate Service Mesh state 'Unmanaged'.
+    ...                The operator will not recreate/update the Service Mesh CR if removed or changed.
+    [Tags]    Operator    Smoke    ODS-2526    ServiceMesh-Unmanaged
```

Comment:
wouldn't this test case risk to break the RHOAI installation and consequently Smoke?

Reply:
What would be a preferable tag? Also, as with many operator tests, component states will need to be tested. Thus, restoring to original state is needed (as this test case does upon teardown). We can move to another tag, and remove it from Smoke if that would be better.

Reply:
I think sanity could be a good compromise